packages/Search/src/DTK_LinearBVH_def.hpp

```diff
@@ -42,9 +40,9 @@ BoundingVolumeHierarchy<DeviceType>::BoundingVolumeHierarchy(
 
     if ( size() == 1 )
     {
-        iota( _indices );
+        Kokkos::View<size_t *, DeviceType> permutation_indices( "dummy", 1 );
```

Comment:
I would not call this "dummy". You do care that `permutation_indices[0] = 0`.

Reply:
Sure.  I can change it.  It is a temporary though.  And yes, the single element has to be zero.